hypha/apply/dashboard/views.py

```diff
@@ -139,172 +163,90 @@ def get(self, request, *args, **kwargs):
 
         # Applications by reviewer
-        my_submissions, my_inactive_submissions = self.get_my_submissions(request, qs)
+        my_submissions, my_inactive_submissions = self.my_submissions(submissions)
 
-        context = {
-            'my_review': my_review,
-            'in_review_count': my_review_qs.count(),
-            'display_more': display_more,
-            'my_reviewed': my_reviewed,
-            'display_more_reviewed': display_more_reviewed,
-            'filter': filterset,
-            'active_statuses_filter': active_statuses_filter,
+        extra_context = {
+            'awaiting_reviews': self.awaiting_reviews(submissions),
+            'my_reviewed': self.my_reviewed(submissions),
             'my_submissions': my_submissions,
             'my_inactive_submissions': my_inactive_submissions,
         }
 
-        return render(request, 'dashboard/reviewer_dashboard.html', context)
-
-    def get_my_reviews(self, user, qs):
-        limit = 5
-        my_review_qs = qs.in_review_for(user).order_by('-submit_time')
-        my_review_table = ReviewerSubmissionsTable(my_review_qs[:limit], prefix='my-review-')
-        display_more = (my_review_qs.count() > limit)
-
-        return my_review_qs, my_review_table, display_more
-
-    def get_my_reviewed(self, request, qs):
-        # Replicating django_filters.views.FilterView
-        my_reviewed_qs = qs.reviewed_by(request.user).order_by('-submit_time')
-
-        kwargs = {
-            'data': request.GET or None,
-            'request': request,
-            'queryset': my_reviewed_qs,
-        }
-        filterset = SubmissionReviewerFilterAndSearch(**kwargs)
-        my_reviewed_qs = filterset.qs
-
-        limit = 5
-        my_reviewed_table = ReviewerSubmissionsTable(my_reviewed_qs[:limit], prefix='my-reviewed-')
-        display_more_reviewed = (my_reviewed_qs.count() > limit)
-
-        return filterset, my_reviewed_qs, my_reviewed_table, display_more_reviewed
-
-    def get_my_submissions(self, request, qs):
-        my_submissions = qs.filter(
-            user=request.user
-        ).active().current().select_related('draft_revision')
-        my_submissions = [
-            submission.from_draft() for submission in my_submissions
-        ]
-
-        my_inactive_submissions_qs = qs.filter(user=self.request.user).inactive().current()
-        my_inactive_submissions_table = ReviewerSubmissionsTable(
-            my_inactive_submissions_qs, prefix='my-submissions-'
-        )
-        return my_submissions, my_inactive_submissions_table
-
-    def get_context_data(self, **kwargs):
-        kwargs = super().get_context_data(**kwargs)
-        search_term = self.request.GET.get('query')
-        kwargs.update(
-            search_term=search_term,
-        )
-
-        return super().get_context_data(**kwargs)
+        current_context = super().get_context_data(**kwargs)
+        return {**current_context, **extra_context}
```

Comment:
This does the same thing as "context.update({…})" does in other classes in this file? I would prefer to use the same pattern in all places. I think I prefer "context.update({…})".